examples/website-demos/social-messaging/src/components/ThreadReply/ThreadReply.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Avatar,
+  MessageRepliesCountButton,
+  useMessageContext,
+} from 'stream-chat-react';
+import { LeftReply, RightReply, } from '../../assets';
+
+import './ThreadReply.scss';
+
+export const ThreadReply: React.FC = () => {
+  const { handleOpenThread, isMyMessage, message, } = useMessageContext();
+
+  const myMessage = isMyMessage();
+
+  const uniqueThreadParticipants = Array.from(new Set(message.thread_participants)).slice(0, 2);
+
+  return (
+    <>
+      {message.thread_participants ? (
+      <div className='message-wrapper-inner-reply'>
+        {!myMessage && (
+          <>
+            <RightReply />
+            {uniqueThreadParticipants[0] && uniqueThreadParticipants.map((user, i) => (
```

Comment:
what happens if a thread has a lot of participants? we should limit to 4 or so

Reply:
line 15 slices the number of participants to 2. I didnt see anything in the design with more than 2. I can increase to 4 if you like that better?